ga4gh/backend.py

```diff
@@ -26,6 +27,16 @@ def __init__(self, dataDir, variantSetClass):
                 self._variantSetIdMap[variantSetId] = variantSetClass(
                     variantSetId, relativePath)
         self._variantSetIds = sorted(self._variantSetIdMap.keys())
+        # self._callSetIdMap map each callSetId to all variantSets
+        # which this callSetId has calls in
+        for variantSetId in self._variantSetIds:
```

Comment:
It would be more natural to use
```python
for variantSet in self._variantSetIdMap.values():
     sampleNames = variantSet.getSampleNames()
     # etc.
```
We should add the method `getSampleNames()` to VariantSet because accessing 'protected' instance variables is a bad idea. (In Python, the convention is that `__instanceVariable` is private to a class and `_instanceVariable` is private to a class and all of its subclasses.)